src/middlewares/authorization.js

```diff
@@ -0,0 +1,15 @@
+import verifyToken from '../utils/authentication';
+
+const checkToken = (req, res, next) => {
+  const token = req.headers['x-access-token'] || req.query.token || req.headers.authorization;
+  if (token) {
+    verifyToken(token);
+    return next();
+  }
+  return res.status(401).json({
+    status: 401,
```

Comment:
Add a `success` object to all responses as discussed during standup today. e.g. `{ success: true}`